index.html

```diff
@@ -182,6 +182,14 @@ <h3>
               The <dfn>current posture</dfn>.
             </td>
           </tr>
+          <tr>
+            <td>
+              <dfn data-dfn-for="Document">[[\OverridePosture]]</dfn>
+            </td>
+            <td>
+              Posture to override posture defined by hardware. Used in testing.
```

Comment:
```suggestion
              Posture to override posture defined by hardware. Used in automation.
```

Reply:
I'd mention the possible values it can have: `"[=continuous=]", "[=folded=]" or null`.

Reply:
Ooops. Found this after pushing https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/9a3c613454ea244a392247beec1ea602fbc8486f. If spec doesn't need to explain null then maybe this sentence style is better than bullets. Or it could also have "or `null`. Set value as `null` to defined posture by hardware."

Reply:
testing => automation fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/daf7abfa604cb53e4245f10209d9ca499b1eae8d